Configuration/AlCa/python/autoAlca.py

```diff
@@ -1,87 +1,40 @@
-
-                  "Express"        : "SiStripCalZeroBias+TkAlMinBias+SiStripPCLHistos+SiStripCalMinBias+SiStripCalMinBiasAAG+Hotline+LumiPixelsMinBias+SiPixelCalZeroBias",
-                  "ExpressCosmics" : "SiStripPCLHistos+SiStripCalZeroBias+TkAlCosmics0T+SiPixelCalZeroBias",
-                  "ExpressAlignment":"TkAlMinBias",
-                  # Used for new PCC PCL introduced in 2018
-                  "ALCALumiPixelsCountsExpress":"AlCaPCCRandom",
-                  # These two cannot run on RAW, they are just meant to run on the dedicated AlcaRAW so they do not enter the allForPrompt list
-                  "AlCaPhiSym"     : "",
-                  "AlCaP0"         : "",
-                  # ---------------------------------------------------------------------------------------------------------------------------
-                  "HcalNZS"        : "HcalCalMinBias",
-                  # This is in the AlCaRecoMatrix, but no RelVals are produced
-                  # 'TestEnablesTracker'  : 'TkAlLAS'
-                  # 'TestEnablesEcalHcal' : 'HcalCalPedestal'
-                  "MET" : "HcalCalNoise",
-                  "SingleMu" : "MuAlCalIsolatedMu+MuAlOverlaps+TkAlMuonIsolated+MuAlZMuMu+HcalCalHO",
-                  "DoubleMu" : "MuAlCalIsolatedMu+MuAlOverlaps+TkAlZMuMu",
-                  "DoubleMuParked" : "MuAlCalIsolatedMu+MuAlOverlaps+TkAlZMuMu",
-                  "MuOniaParked" : "TkAlJpsiMuMu+TkAlUpsilonMuMu",
-                  "DoubleElectron" : "EcalCalZElectron+EcalUncalZElectron",
-                  "StreamExpress" : "SiStripCalZeroBias+TkAlMinBias+SiStripPCLHistos+SiStripCalMinBias+SiStripCalMinBiasAAG+Hotline+LumiPixelsMinBias+SiPixelCalZeroBias+SiPixelCalSingleMuon+PPSCalTrackBasedSel",
-                  "StreamExpressHI" : "SiStripCalZeroBias+TkAlMinBiasHI+SiStripPCLHistos+SiStripCalMinBias+SiStripCalMinBiasAAG+SiPixelCalZeroBias"
+AlCaRecoMatrix = {
+                  "ALCALumiPixelsCountsExpress" : "AlCaPCCRandom",
+                  "AlCaLumiPixelsCountsPrompt"  : "AlCaPCCZeroBias+RawPCCProducer",
+                  # These two (AlCaPhiSym, AlCaP0) cannot run on RAW, they are just meant to run on the dedicated AlcaRAW so they do not enter the allForPrompt list
+                  "AlCaPhiSym"                  : "",
+                  "AlCaP0"                      : "",
+                  "ALCAPPS"                     : "PPSCalMaxTracks", # Express producer
+                  "AlCaPPS"                     : "PPSCalMaxTracks", # Prompt  producer
+                  "Charmonium"                  : "TkAlJpsiMuMu",
+                  "Commissioning"               : "HcalCalIsoTrk+HcalCalIsolatedBunchSelector+TkAlMinBias+SiStripCalMinBias",
+                  "Cosmics"                     : "SiPixelCalCosmics+SiStripCalCosmics+TkAlCosmics0T+MuAlGlobalCosmics",
+                  "DoubleEG"                    : "EcalCalZElectron+EcalUncalZElectron+HcalCalIterativePhiSym+HcalCalIsoTrkProducerFilter",
```

Comment:
we dont have `EcalCalZElectron` in the twiki

Reply:
on the other hand `EcalUncalWElectron` is  attached to`DoubleEG` in the twiki

Reply:
fixed

Reply:
just FYI `EcalCalZElectron` is not present in the tier0 matrix, but until we get a confirmation from ECAL wethere this is to be kept or not, I don't see any problem in leaving it here for the moment.